.yamato/project.metafile

```diff
@@ -7,7 +7,6 @@ test_editors:
   - 2021.1
   - 2021.2
   - 2020.3
-  - 2019.4
```

Comment:
um, we want to disable 2019 validation?

Reply:
Yes, this was called out in the Netcode slack :D that since UTP doesn't support 2019.4 and fails all 2019.4 we would up the min spec to 20.3